src/MainView.vue

```diff
@@ -1,46 +1,47 @@
 <template>
+  <!-- 
+    If you prepend an attribute with `:` (short for `v-bind:`), vue will treat the attribute's
+    value as javascript. This loads the imageUrl as imported in the below `script` section.
+  -->
+  <img :src="imageUrl" alt="CloudBolt logo" />
```

Comment:
this has no purpose other than having the image in its previous position was bothering me, not a necessary change